plugins/usertiming.js

```diff
@@ -87,21 +93,49 @@
 		options: {"from": 0, "window": BOOMR.window},
 
 		/*
+		 * Gets the user timings, filters out those that have already been sent
 		 * Calls the UserTimingCompression library to get the compressed UserTiming
-		 * data that occurred since the last call.
+		 * data that occurred since the last call
 		 *
-		 * @returns {string} Compressed UserTiming data
+		 * @returns {string} UserTiming data
 		 */
 		getUserTiming: function() {
-			var timings, res,
-			    now = BOOMR.hrNow(),  // Get a timestamp to compare with performance marks and measures.
-			    utc = window.UserTimingCompression || BOOMR.window.UserTimingCompression;
+			var frame, entries, options;
```

Comment:
I think we could simplify this (realizing it was migrated from https://github.com/nicjansma/usertiming-compression.js/blob/master/src/usertiming-compression.js#L381) to just using the `impl.options` always and not having a local `options`.